contracts/modules/Checkpoint/EtherDividendCheckpoint.sol

```diff
@@ -197,7 +182,7 @@ contract EtherDividendCheckpoint is DividendCheckpoint {
         Dividend storage dividend = dividends[_dividendIndex];
         dividend.reclaimed = true;
         uint256 remainingAmount = dividend.amount.sub(dividend.claimedAmount);
-        address owner = IOwnable(securityToken).owner();
+        address payable owner = address(uint160(IOwnable(securityToken).owner()));

```

Comment:
Can we cast directly?

Reply:
We can't directly cast non-payable to payable but owner is already being returned as payable in the interface so we don't need to cast it here. I'll make the change.

If we make owner non-payable in the interface, we'll have to use this casting here.